### PR TITLE
Fix: display correctly month if languageCode ne EN

### PR DESCRIPTION
**Pull request by hucste**
if i18n exists, the month into this list item is not correctly displayed. Actually, only in english.
With this fixe, month is displayed segun language code.

### CODE REVIEW

**Comment by reuixiy** [comment]
So as the comment in config.toml,

https://github.com/reuixiy/hugo-theme-meme/blob/a7c47aa5f4f33373068a24fb9f735a2b8dfee501/config-examples/en/config.toml#L449-L450

I don't think this is a bug. Though, it could be an enhancement.

Also, in your code, if `i18nMonth` is enabled, then the `listDateFormat` will not be respected -- it's hard-coded to `"January 2"` format. But there are many different date formats in different countries, see:

- https://en.wikipedia.org/wiki/Date_format_by_country
- https://gohugo.io/functions/format/

In other words, the character is `i18n`ed but the format is not, or is destroyed.

**Comment by hucste** [comment]
Ok, I understand the problem. For the moment, I don't known how fix it.
but you have to admit that having the dates, once in FR (or other lang), once in EN, so sucks. :(
